python/tank/deploy/tank_commands/install.py

```diff
@@ -366,6 +427,13 @@ def run_interactive(self, log, args):
             log.info("For example, to install the houdini engine into the Asset environment:")
             log.info("> tank install_engine Asset tk-houdini")
             log.info("")
+            log.info("Comment and structure preserving mode")
```

Comment:
i didn't pay much attention to it since it's a temp thing and will be going away soon anyways!